sale_commission/stock_picking.py

```diff
@@ -48,7 +48,7 @@ def _invoice_line_hook(self, cr, uid, move_line, invoice_line_id, context=None):
         if context is None:
             context = {}
         agent_pool = self.pool.get('invoice.line.agent')
-        super(stock_picking, self)._invoice_line_hook(cr, uid, move_line, invoice_line_id, context=context)
+        super(stock_picking, self)._invoice_line_hook(cr, uid, move_line, invoice_line_id)
```

Comment:
Hey,

_invoice_line_hook doesn't take context as argument anymore in fact. Without this patch, the tests of commission fails